Add license file

AjanZhong opened the pull request:
Add LICENSE file which points to license file resides in source code.

Tracked-On: OAM-106807

sysopenci reviewed (commented):
Failure when executing preintegration builder

FAILURE:CheckBug Bad comments/Bugs

More details, logs, and binaries: /absp/builders/celadon-preintegration/builds/4920

sysopenci reviewed (commented):
Failure when executing preintegration builder

FAILURE:CheckBug Bad comments/Bugs

More details, logs, and binaries: /absp/builders/celadon-preintegration/builds/4923

sysopenci reviewed (commented):
PR integrated into 'preintegration' builder



More details, logs, and binaries: /absp/builders/celadon-preintegration/builds/4924

Done by "Zhong, Fangjian" <fangjian.zhong@intel.com>

sysopenci reviewed (commented):
PR integrated into 'premerge' builder



More details, logs, and binaries: /absp/builders/celadon-premerge/builds/2841

Done by "Zhong, Fangjian" <fangjian.zhong@intel.com>

Commit:
Add license file

Add LICENSE file which points to license file resides in source code.

Tracked-On: OAM-106807
Signed-off-by: Zhong,Fangjian <fangjian.zhong@intel.com>

sysopenci reviewed (commented):
cactus finished building and testing your changes 

It was requested for merge by "Zhong, Fangjian" <fangjian.zhong@intel.com> 

so we merge it. Thanks for your contribution.

more details, logs, and binaries: /absp/builders/celadon-mergerequest/builds/3240
